### ISSUE TITLE
Support for customer allocation policy

### ISSUE THREAD

**Issue by tomkerkhove** [Azure/azure-iot-cli-extension#200]
Are there plans to support using custom allocation policy when creating a new enrollment group in Device Provisioning Service?

If it's already there, sorry that I've missed it in https://docs.microsoft.com/en-us/cli/azure/ext/azure-iot/iot/dps/enrollment-group?view=azure-cli-latest#ext-azure-iot-az-iot-dps-enrollment-group-create-optional-parameters

**Expected behavior**
Be able to use following command:
`az iot  dps enrollment-group create --dps-name $name --resource-group $resourceGroupName --enrollment-id $enrollmentGroupId --allocation-policy custom`

It would require more information ofcourse but open for anything.

**Environment (please complete the following information):**

- OS: Windows 10, Ubuntu 16.04
- Shell: powershell
- Az CLI version: 2.5.1
- IoT extension version: 0.8.9

```json
{
  "azure-cli": "2.5.1",
  "azure-cli-command-modules-nspkg": "2.0.3",
  "azure-cli-core": "2.5.1",
  "azure-cli-nspkg": "3.0.4",
  "azure-cli-telemetry": "1.0.4",
  "extensions": {
    "azure-cli-iot-ext": "0.8.9",
    "azure-devops": "0.13.0"
  }
}
```

**Comment by tomkerkhove** [comment]
Those looking for a workaround can use PowerShell - https://github.com/Azure/azure-powershell/blob/master/src/DeviceProvisioningServices/DeviceProvisioningServices/help/Add-AzIoTDeviceProvisioningServiceEnrollment.md#add-aziotdeviceprovisioningserviceenrollment

**Comment by digimaun** [comment]
@tomkerkhove thank you for your patience, the latest released `0.9.7` should include the `custom` allocation policy.

Also as an FYI, `azure-cli-iot-ext` is deprecated -- you should be using `azure-iot` at all times.

The docs [you linked](https://docs.microsoft.com/en-us/cli/azure/ext/azure-iot/iot/dps/enrollment-group?view=azure-cli-latest#ext-azure-iot-az-iot-dps-enrollment-group-create-optional-parameters) have been updated as well including a usage example.

**Comment by tomkerkhove** [comment]
Thanks for the update!
